src/public/js/at-sce.util.js

```diff
@@ -42,6 +42,81 @@ $(function () {
 });
 
 /* Calculator Functions */
+function SumFixedCost() {
+  var rent = document.getElementById("rent").value;
```

Comment:
You could do all of this initialization at the beginning of the file as constants since this elements won't change and you'll only use the methods on them to get data and assign values, don't think there's need to do it on every function.

Reply:
`const RENT = document.getElementById("rent").value;`